src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,18 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class CEO {
+    String name;
+    ArrayList<Manager> managerList;
```

Comment:
Jeszcze brakuje modyfikatorów te pola powinny być private a jeśli potrzebowałabyś dostęp do nich to powinien on zostać zrealizowany przez getter